packages/stores/src/account/msg/amino.ts

```diff
@@ -0,0 +1,115 @@
+import { Msg } from "@cosmjs/launchpad";
```

Comment:
We should keep in mind that we'll have to refactor this file before merging CL or multiple wallets. These amino files are no longer required thanks to the `proto-codecs` package

Reply:
Yeah I refactored the swap messages (added one, removed one, refactored 2). I imagine we'll have some nasty conflicts there.